finality-aleph/Cargo.toml

```diff
@@ -14,6 +14,7 @@ current-aleph-bft-rmc = { package = "aleph-bft-rmc", version = "0.6" }
 legacy-aleph-bft = { package = "aleph-bft", version = "0.19" }
 legacy-aleph-bft-rmc = { package = "aleph-bft-rmc", version = "0.5" }
 
+aleph-clique = { package = "clique-network", path = "../clique" }
```

Comment:
I think importing it just as `clique-network` might be slightly better anyway, there is nothing _that_ Aleph-specific there.

Reply:
Fixed with `network-clique` name